bigtable/client/rpc_backoff_policy.cc

```diff
@@ -46,12 +46,18 @@ void ExponentialBackoffPolicy::setup(grpc::ClientContext& /*unused*/) const {}
 std::chrono::milliseconds ExponentialBackoffPolicy::on_completion(
     grpc::Status const& status) {
   using namespace std::chrono;
-  // TODO(#57) - we also need to randomize the sleep period.
   auto delay = duration_cast<milliseconds>(current_delay_);
-  current_delay_ *= 2;
-  if (current_delay_ >= maximum_delay_) {
-    current_delay_ = maximum_delay_;
+  std::uniform_int_distribution<int> rng_distribution(
```

Comment:
Per our discussion online, this function is returning a constant value after `current_delay_ == maximum_delay_`.  We want random values with increasingly large ranges.

Reply:
Done.